app/views/mentors/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Mentors</h1>
+
```

Comment:
can we delete ```<h1>Mentors</h1>``` please. We have a standard page title in the nav bar. I asked to add ```@page_title``` in the index action. just do a project search to find the @page_title so you can see how it works